xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -2647,6 +2648,37 @@ absl::Status IrEmitterUnnested::EmitHloInstruction(
               streams.destination_stream_id, streams.source_stream_id));
           return EmitFusion(Cast<HloFusionInstruction>(wrapped));
         }
+        case HloOpcode::kCall: {
+          const ExecutionStreamAssignment& stream_assignment =
+              ir_emitter_context_->execution_stream_assignment();
+          TF_ASSIGN_OR_RETURN(
+              auto stream, stream_assignment.GetSyncExecutionStreamId(wrapped));
+          auto ir_emitter = IrEmitterUnnested::Create(ir_emitter_context_);
+          DCHECK_EQ(wrapped->called_computations().size(), 1);
+          auto computation = wrapped->called_computations().front();
+          TF_RETURN_IF_ERROR(ir_emitter->EmitHloComputation(computation));
+          std::unique_ptr<SequentialThunk> thunk_sequence =
+              ir_emitter->ConsumeThunkSequence();
+          for (auto& thunk : thunk_sequence->thunks()) {
```

Comment:
Can you leave a comment explaining why we need to set this on both the sequential thunk and its contents? Just an API quirk?